tox.ini

```diff
@@ -9,4 +9,4 @@ deps =
 [testenv:lint]
 deps =
     flake8==3.7.7
-commands=flake8 whatthepatch tests setup.py
+commands=flake8 --max-line-length=100 whatthepatch tests setup.py
```

Comment:
I'd leave this at the default

Reply:
Generally disagree (especially for tests), but the default is probably fine now with other mentioned changes.